packages/sdk-components-react/src/xml-element.tsx

```diff
@@ -0,0 +1,32 @@
+import { componentAttribute } from "@webstudio-is/react-sdk";
+import { Children, forwardRef, type ElementRef, type ReactNode } from "react";
+
+export const defaultTag = "div";
+
+// We don't want to enable all tags because Box is usually a container and we have specific components for many tags.
+type Props = {
+  tag: string;
+  children: ReactNode;
+} & Record<typeof componentAttribute, string>;
+
+export const XmlElement = forwardRef<ElementRef<"div">, Props>(
+  ({ tag = "element", children, ...props }, ref) => {
+    const isTextChild = Children.toArray(children).every(
+      (child) => typeof child === "string"
+    );
+
+    const elementName = tag.replaceAll(/[^\p{L}]+/gu, "");
```

Comment:
can't we use browsers own xml parser?

Reply:
for example aren't -_ or . allowed?

Reply:
probably can't sanitize with the dom, only parse ...

Reply:
found something https://github.com/MylesBorins/xml-sanitizer/blob/main/index.js

Reply:
Improved